07LessonSeven/homework/index.html

```diff
@@ -5,6 +5,34 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+<header>
+  <h1>Professional Portal</h1>
+</header>
+  <nav>
+<div>
+<a href=""><span>home</span></a>
+<a href=""><span>Companies</span></a>
+<a href=""><span>Positions</span> </a>
+</div>
+
```

Comment:
Indentation also applies to all content of a parent container. The `<div>` and all `<a>` tags should be indented.